src/components/icons/icons.test.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+
+import IonIcon from './icons';
+import { IonIconProps } from './icons';
+import { iconType } from './svgs/icons';
+
+const defaultIcon: IonIconProps = {
+  type: 'pencil',
+};
+
+const sut = (props = defaultIcon) => render(<IonIcon {...props} />);
+const getIcon = (type: iconType) => screen.getByTestId(`ion-icon-${type}`);
+
+describe('IonIcon', () => {
+  it('should render default icon of type pencil', () => {
+    sut();
+    const icon = getIcon('pencil');
```

Comment:
```suggestion
    const icon = getIcon(defaultIcon.type);
```

use in everywhere that you call pencil